test/conformance/parallel/parallel_suite_test.go

```diff
@@ -45,17 +51,52 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	testclient.Client = testclient.New("")
 	Expect(testclient.Client).NotTo(BeNil())
 
+	// discovers valid ptp configurations based on clock type
 	err := testconfig.CreatePtpConfigurations()
 	if err != nil {
 		Fail(fmt.Sprintf("Could not create a ptp config, err=%s", err))
 	}
-
+	By("Refreshing configuration", func() {
+		ptphelper.WaitForPtpDaemonToBeReady()
+		fullConfig = testconfig.GetFullDiscoveredConfig(pkg.PtpLinuxDaemonNamespace, true)
+	})
 	ptphelper.RestartPTPDaemon()
```

Comment:
will this restart Daemon affect any other parallel tests for an example CPU usage etc 
or is this happens at the beginning of test case ?